module/zfs/vdev_queue.c

```diff
@@ -631,7 +631,6 @@ vdev_queue_aggregate(vdev_queue_t *vq, zio_t *zio)
 		return (NULL);
 
 	size = IO_SPAN(first, last);
-	ASSERT3U(size, <=, limit);
```

Comment:
We should drop this anyway since it's redundant with the VERIFY checks in `abd_alloc_linear()` / `abd_alloc()` which more correctly check that the size is less than SPA_MAXBLOCKSIZE.